cobigen-cli/src/main/resources/pom.xml

```diff
@@ -63,7 +63,12 @@
         <artifactId>jsonplugin</artifactId>
         <version>2.0.0</version>
       </dependency>
-          <dependency>
+      <dependency>
+        <groupId>com.devonfw.cobigen</groupId>
+        <artifactId>templates-devon4j</artifactId>
+        <version>3.1.6</version>
+      </dependency>
+    <dependency>
```

Comment:
Can you change the `tsPlugin` version to `2.3.1`? I have just released it today.